components/eamxx/src/physics/mam/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 add_library(mam eamxx_mam_microphysics_process_interface.cpp)
 target_compile_definitions(mam PUBLIC EAMXX_HAS_MAM)
 add_dependencies(mam mam4xx_proj)
-target_include_directories(mam PRIVATE
+target_include_directories(mam PUBLIC
```

Comment:
Since you link mam4xx and haero below, do you really need these includes here?

Reply:
I just removed this line and the headers weren't found, so evidently we do need them.

Reply:
It's obv a minor thing, so not really important. But I suspect the problem is with the imported target, which do not store any information regarding include dirs. If [here](https://github.com/E3SM-Project/scream/blob/580beae6b310c11c0f34ba9041ae63fdda99bbac/components/eamxx/tpls/CMakeLists.txt#L72) and [here](https://github.com/E3SM-Project/scream/blob/580beae6b310c11c0f34ba9041ae63fdda99bbac/components/eamxx/tpls/CMakeLists.txt#L97) you also do something like
```
target_include_directories (mam4xx PUBLIC ${MAM4XX_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR})
```
then you don't need these. Though you may argue it's a potato potatoe kind of difference...